presto-main/src/main/java/io/prestosql/sql/planner/optimizations/UnaliasSymbolReferences.java

```diff
@@ -203,7 +203,7 @@ public PlanAndMappings visitExplainAnalyze(ExplainAnalyzeNode node, UnaliasConte
             Symbol newOutputSymbol = mapper.map(node.getOutputSymbol());
 
             return new PlanAndMappings(
-                    new ExplainAnalyzeNode(node.getId(), rewrittenSource.getRoot(), newOutputSymbol, node.isVerbose()),
+                    new ExplainAnalyzeNode(node.getId(), rewrittenSource.getRoot(), newOutputSymbol, mapper.map(node.getActualOutputs()), node.isVerbose()),
```

Comment:
The call to `mapper.map()` potentially modifies `mapping`. For clarity, move it before the constructor, like mapping of the output symbols.

Reply:
Moved it !!